modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-suma.adoc

```diff
@@ -213,7 +229,99 @@ transactional-update
 
 . Reboot.
 
-=== Configure Custom Persistent Storage
+[IMPORTANT]
+====
+{sle-micro} is designed to update itself automatically by default and will reboot after applying updates.
+However, this behavior is not desirable for the {productname} environment.
+To prevent automatic updates on your server, {productname} disables the transactional-update timer during the bootstrap process.
+
+If you prefer the {sle-micro} default behavior, enable the timer by running the following command:
+
+[source, shell]
+----
+systemctl enable --now transactional-update.timer
+----
+====
+
+To continue with deployment, see xref:installation-and-upgrade:container-deployment/suma/server-deployment-suma.adoc#deploy-suma-server-persistent-storage[].
+
+
+[[deploy-suma-server-sles]]
+=== Deploy on {sles} {bci-suma}
+
+Alternatively, you can deploy {productname} on {sles} {bci-suma}.
+
+
+The following procedure describes the main steps of the installation process.
+
+.Procedure: Installing {productname} on {sles} {bci-suma}
+. Locate and download {sles} {bci-suma} [literal]``.iso`` at https://www.suse.com/download/sles/.
+// FIXME Do we need the following step?
++
+. Make sure that the {scc} code you plan to use has the Extensions enabled.
+. Start the installation of {sles} {bci-suma}.
+  .. On the [literal]``Language, keyboard and product selection`` select the product to install.
+  .. On the [literal]``License agreement`` read the agreement and check [guimenu]``I Agree to the License Terms``.
+. Select the registration method.
+
++
+
+[[NOTE]]
+====
+Please note that for {sles} {bci-suma}, you are required to have a valid {sles} subscription and corresponding regcode, which you must provide on this screen.
+You will be required to enter the {productname} Extension regcode below.
+====
+. In the screen [literal]``Extensions and Modules Selection`` check the following:
++
+
+  * {productname} Server Extension
+  * Basesystem Module
+  * Containers Module
++
+
+Enter the {productname} Extension regcode.
+. Complete the installation.
+. When the installation completes, log in to the newly installed server as root.
+. Update the System (optional, if the system was not set to
+download updates during install):
+
++
+
+[source,shell]
+----
+zypper up
+----
+
++
+
+// FIXME: where is the following "Reboot" actually required?
+
+. Reboot
+. Log in as root and install podman
+
++
+
+[source,shell]
+----
+zypper install podman
```

Comment:
```suggestion
zypper install podman mgradm mgradm-bash-completion
```
I think we are missing `mgradm` and `mgradm-bash-completion`, was we have for `mgrpxy` for the proxy. Otherwise, we cannot continue later with deploying SUMA/MLM with `mgradm`.
See https://github.com/uyuni-project/uyuni-docs/pull/3834/files#diff-183022748e3c5a3480fb8789a0c5b574382f2f7a051537ca50ab2ca2edbbfa4eR220

Reply:
I did a test with `zypper se mgradm` and apparently those are already installed. So not sure if we then need them.

Reply:
We should maybe add a note in there that they get automatically installed but if not, do this.

Reply:
This is also a good solution.

Reply:
I think they were not automatically with an earlier release or MU. A note or remark as abid proposes sounds good.  Maybe, we can relax that statement later.